docs/wip/traffic/permissive_traffic_policy_mode.md

```diff
@@ -0,0 +1,343 @@
+---
+title: "Permissive traffic policy mode"
+description: "Permissive traffic policy mode"
+type: docs
+aliases: ["permissive_traffic_policy_mode.md"]
+---
+
+# Permissive traffic policy mode
+Permissive traffic policy mode in OSM is a mode where [SMI][1] traffic policy enforcement is bypassed. In this mode, OSM automatically discovers services that are a part of the service mesh and programs traffic policy rules on each Envoy proxy sidecar to be able to communicate with these services.
+
+## When to use permissive traffic policy mode
+Since permissive traffic policy mode bypasses [SMI][1] traffic policy enforcement, it is suitable for use when connectivity between applications within the service mesh should flow as before the applications were enrolled into the mesh. This mode is suitable in environments
+where explicitly defining traffic policies for connectivity between applications it not feasible.
```

Comment:
```suggestion
where defining traffic policies  explicitly for connectivity between applications it not feasible.
```

Reply:
I skipped this suggestion since it seemed unnecessary.